### PR TITLE
Reorganise the Android manual

### PR DESCRIPTION
**Pull request by settermjd**
Reorganise the Android documentation so that it's easier to maintain, and easier to navigate and use for the user

### CODE REVIEW

File: user_manual/functionality.rst [hunk 1]
```diff
@@ -125,35 +94,35 @@ Enter a new password, and click *"Connect"*, and the password will be updated.
 
 If you want extra security, please refer to the `Passcode Locks & Pins`_ section.
 
-Files
------
+Manage Files
+------------
 
-All Files
```
Comment (davigonz) [inline user_manual/functionality.rst]: **All files** link seems to be broken, if you click on it nothing happens

File: user_manual/security.rst [hunk 2]
```diff
@@ -0,0 +1,18 @@
+Security
+========
+
+Passcode Locks & Pins
+---------------------
+
+You can also set a passcode lock to further protect your files and folders.
```
Comment (davigonz) [inline user_manual/security.rst]: Fingerprint should be included here as well.

**Comment by davigonz** [comment]
@settermjd thanks for reorganising this, I like the new distribution, can you have a look at the minor changes I requested and solve the conflicts? Thanks

File: user_manual/security.rst (hunk 2 above)
Comment (settermjd) [reply user_manual/security.rst]: Just added it, @davigonz. Can you check that it's correct?

**Comment by davigonz** [comment]
@settermjd it seems there's some conflicts in `user_manual/android_app.rst`. Please solve them so we can merge it.

**Comment by settermjd** [comment]
@davigonz, done.

**Comment by settermjd** [comment]
Closing, as it's well and truly out of date.

**Pull request closed without merging**